Header header
ColorBlobInfo[] color_blobs
